src/App.vue

```diff
@@ -1,10 +1,12 @@
 // App.vue
 <template>
-  <div id="app">
-    <ToDos v-bind:todoEntries="todoEntries" @delete-todo-event="deleteTodDoItem" />
-    <AddToDoButton @add-todo-event="addToDoItem" />
+  <div class="app">
+    <div class="headline1">
+      <h1 class="headline"><i class="bi bi-check-square-fill"></i>My Todo-s</h1></div>
```

Comment:
Programmieren ist mehr lesen, als schreiben. 

Deswegen sollte man immer darauf achten, dass man den code auch gut formattiert: 
Der closing tag (also in diesem Fall der </div> sollte in eine neue Zeile.